index.html

```diff
@@ -8,21 +8,51 @@
     <title>Awsome Books</title>

```

Comment:
- [ ] **Please add the current date to your website. It is a mandatory project requirement.**

Following is the requirement from [project description](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md#project-requirements)

_Display the current date below the navigation bar. For this requirement, you should use a library - Luxon._

Please make sure to add the current date on your website. You have to use the Luxon library to implement this.